tests/integration/targets/aci_epg_useg_attribute/tasks/vm_custom_attr.yml

```diff
@@ -0,0 +1,204 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Christian Kolrep (@Christian-Kolrep)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
```

Comment:
I think the setting vars, query system, verify cloud, clean environment ( tenant, ap, bd, creation etc ) can be done in main.yaml.

The for each attr type, you would define the tests for that type and clean up all the created objects in that test file. Then the next type would have clean environment again.

Finally in main.yaml you can then delete the tenant, and thus all the generic configuration.

Reply:
I try to remember but i tried this with include and import. The problem was always errors with missing vars from tenant, ap, bd, creation etc. Maybe you have an idea how it is done correctly?